### DISCUSSION TITLE
Large Paraview vtk ASCII files; How about binary output?

### DISCUSSION THREAD

**Discussion by kengwit** [OpenRadioss/OpenRadioss#2273] [Q&A]
Hi,

I performed a test analysis on the Camry car model crash found here:

https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/10518559/Camry+Impact+Model+in+LS-DYNA+format

Upon completion, I used the supplied converter anim_to_vtk to generate vtk files, but the resulting vtk files are huge, significantly slowing Paraview down. Is there plan to generate binary output (instead of ascii), or better still, generate output that can be read by LS-PrePost (d3plot)? 

Thanks
Kengwit

**Comment by SyazwanSamad** [comment]
Hi @kengwit 

Perhaps you can head over to this discussion. https://github.com/orgs/OpenRadioss/discussions/2361
There's a Python tool to convert anim to d3plot.  

Regards
Syazwan
